### ISSUE TITLE
Was does 3rd argument 'title' into getQRCodeGoogleUrl do?

### ISSUE THREAD

**Issue by nodesocket** [PHPGangsta/GoogleAuthenticator#70]
I don't see the title showing up in Authy. Also, I can't seem to pass any special characters such as `.` or `-` into the first argument `name` as well.

**Comment by baris1892** [comment]
The `title` will be shown in the Google Authenticator app before the URL (`$name` parameter).  


Example: 
`$ga->getQRCodeGoogleUrl("example.com", $secret, "Title Test")`
will be displayed in the Google Authenticator app as follows:
`Title Test (example.com)`

**Comment by nunia** [comment]
yes, first the title will be shown then it will echo the URL link
